app/models/membership.rb

```diff
@@ -19,6 +21,13 @@ def try_destroy
 
   private
 
+  def admin_must_have_active_invitation_status
```

Comment:
what if I invite another admin, and he did not yet confirm the invitation?

Reply:
fixed now. thanks for catching that!